letsencrypt-apache/letsencrypt_apache/parser.py

```diff
@@ -56,9 +58,11 @@ def __init__(self, aug, root, ctl):
         # Set up rest of locations
         self.loc.update(self._set_locations())
 
-        # Must also attempt to parse sites-available or equivalent
-        # Sites-available is not included naturally in configuration
-        self._parse_file(os.path.join(self.root, "sites-available") + "/*")
+        # Take the CentOS layout into account, httpd.conf not in httpd root
+        self._parse_file(os.path.join(self.root, "conf") + "/httpd.conf")
```

Comment:
The above line is redundant.  In line `45` the `self._parse_file(self.loc["root"])` call already parses `conf/httpd.conf` if run under CentOS. (See line `553`).